mypy/server/astmerge.py

```diff
@@ -360,7 +360,8 @@ def visit_callable_type(self, typ: CallableType) -> None:
     def visit_overloaded(self, t: Overloaded) -> None:
         for item in t.items():
             item.accept(self)
-        t.fallback.accept(self)
+        if t.fallback is not None:
```

Comment:
When can this be `None`? It sounds like a potential bug, and cause trouble elsewhere. Would it be possible to ensure that `fallback` always gets initialized with a non-`None` value before we reach AST merge? At least add a TODO comment.

Reply:
Few lines above there is a comment about `CallableType.fallback`, I think it equally applies to `Overloaded`. Or you want me to add a similar comment here?

Reply:
That sounds reasonable.